OptimizelySDK.Tests/NotificationTests/NotificationCenterTests.cs

```diff
@@ -0,0 +1,225 @@
+﻿/* 
+ * Copyright 2017, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using Moq;
+using NUnit.Framework;
+using OptimizelySDK.Entity;
+using OptimizelySDK.ErrorHandler;
+using OptimizelySDK.Event;
+using OptimizelySDK.Logger;
+using OptimizelySDK.Notifications;
+
+namespace OptimizelySDK.Tests.NotificationTests
+{
+    public class NotificationCenterTests
+    {
+        private Mock<ILogger> LoggerMock;
+        private NotificationCenter NotificationCenter;
+        private TestNotificationCallbacks TestNotificationCallbacks;
+        private NotificationCenter.NotificationType NotificationTypeActivate = NotificationCenter.NotificationType.Activate;
+        private NotificationCenter.NotificationType NotificationTypeTrack = NotificationCenter.NotificationType.Track;
+        private NotificationCenter.NotificationType NotificationTypeFeatureExperiment = NotificationCenter.NotificationType.FeatureExperiment;
```

Comment:
There is a fourth non-mobile NotificationType enum value FeatureRollout beyond Activate, Track, FeatureExperiment .  I do not see FeatureRollout being tested in NotificationCenterTests.cs .

Reply:
Please check here. 
https://github.com/optimizely/csharp-sdk/pull/35/files#diff-8660e0a3c8615b111f08b9e4ab4f9b36R1689
, I have added more tests, you can see in the latest check-in.